app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoFragment.kt

```diff
@@ -40,33 +40,45 @@ class PocketVideoFragment : Fragment() {
         const val FRAGMENT_TAG = "pocket"
     }
 
+    private val compositeDisposable = CompositeDisposable()
+
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
-        val layout = inflater.inflate(R.layout.fragment_pocket_video, container, false)
+        return inflater.inflate(R.layout.fragment_pocket_video, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        pocketHelpButton.setOnClickListener { _ ->
```

Comment:
Have we always had pocket help button?! 😮

Reply:
Yeah, but it's pretty hidden....